src/xz/mytime.c

```diff
@@ -12,7 +12,7 @@
 
 #include "private.h"
 
-#ifdef HAVE_CLOCK_GETTIME
```

Comment:
`mytime_now()` doesn't need to be super precise and it doesn't need to be relative to any known point in time. Unless I'm missing something, `return GetTickCount64();` should be all that is needed to implement `mytime_now()`. The only downside is that it's not available on ancient Windows versions but at some point pre-Vista support likely could be removed anyway to simplify the code a little.

Reply:
Fixed in a new commit. Simply use `GetTickCount64()` now.